Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutEditController.java

```diff
@@ -0,0 +1,40 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+
+/**
+ * This controller handles requests for editing project's public website's about page.
+ * 
+ * @author Rajat Aggarwal
+ *
+ */
+
+@Controller
+public class WebsiteAboutEditController {
+
+    @Autowired
+    private IRetrieveProjectManager projectManager;
+
+    @RequestMapping(value = "sites/{ProjectUnixName}/EditAbout", method = RequestMethod.GET)
```

Comment:
Same as for the JSP, the page should not be accessible through the "sites" url and by anyone. It should be under "auth/..." and only project admins and owners should be able to access it.